### PR TITLE
Docs: Update CKEditor 4 migration documentation after the release of CKE4.15.

### PR DESCRIPTION
**Pull request by godai78**
### Migration guide update.

Closes cksource/content-documentation-proofread#314

Closes cksource/content-documentation-proofread#289

New feature added, deprecated configuration options removed (not used anywhere outside the migration document).

### CODE REVIEW

**Comment by AnnaTomanek** [comment]
This should be targeted at `stable` as it concerns an already released version.

**Comment by AnnaTomanek** [comment]
2\. It looks like you missed quite a few new configuration options. Can you go to https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html, search "4.15" on this page and add all found items?

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]
There are still plenty of new options that were not covered.

File: docs/builds/guides/migrate.md [hunk 1]
```diff
@@ -327,6 +331,14 @@ Note: The number of options was reduced on purpose. We understood that configuri
 		<tr>
 			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-entities">entities</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-entities_additional">entities_additional</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-entities_greek">entities_greek</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-entities_latin">entities_latin</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-entities_processNumerical">entities_processNumerical</a></td>
 			<td>N/A</td>
+		</tr>
+				<tr>
+			<td>
+			<a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_fileName">exportPdf_fileName</a> <br><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_options">exportPdf_options</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_service">exportPdf_service</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_stylesheet">exportPdf_stylesheet</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_tokenUrl">exportPdf_tokenUrl</a> <br>
```
Comment (AnnaTomanek) [inline docs/builds/guides/migrate.md:337]:
```suggestion
			<a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_fileName">exportPdf_fileName</a> <br><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_options">exportPdf_options</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_service">exportPdf_service</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_stylesheets">exportPdf_stylesheets</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-exportPdf_tokenUrl">exportPdf_tokenUrl</a> <br>
```

File: docs/builds/guides/migrate.md [hunk 2]
```diff
@@ -327,6 +331,14 @@ Note: The number of options was reduced on purpose. We understood that configuri
...
+			</td>
+			<td>
+				Refer to the {@link features/export-pdf Export to PDF feature} guide to learn more about about configuring PDF converter in CKEditor 5.
```
Comment (AnnaTomanek) [inline docs/builds/guides/migrate.md:340]:
```suggestion
				Refer to the {@link features/export-pdf Export to PDF feature} guide to learn more about about configuring the HTML to PDF converter in CKEditor 5.
```

**Review by AnnaTomanek — APPROVED** [review]
Looks good now!

**Comment by harleenarora** [comment]
How to migration CKEditor 4 data in CKE4.15?

**Pull request merged**